### PR TITLE
update Sklearn Basic 3 to illustrate entities

### LINKED ISSUE #1: illustrate 1:1 relations in tutorial "Sklearn Basics 3"
Extend the multitable dataset presented in "Sklearn Basics 3: Train a Classifier on a Snowflake Multi-Table Dataset" by adding the entity table "Places".

### PR DESCRIPTION
**Pull request by nairbenrekia**

### CODE REVIEW

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb [hunk 1]
```diff
@@ -101,14 +109,16 @@
...
     "\n",
     "To describe relations between tables, the field `relations` must be added to the dictionary of table specifications. This field\n",
-    "is a list of pairs of tables of the form \n",
-    "```\n",
-    "(<parent table name>, <child table name>)\n",
-    "``` \n"
+    "contains a list of tuples describing the relations between tables. The first two values (Strings) of each tuple correspond to names of both the parent and the child table involved in the relation. A third value (Boolean) can be optionally added to the tuple to indicate if the relation is either ``1:n`` or ``1:1`` (entity). For example, If the tuple ``(table1, table2, True)`` is contained in this field, it means that:\n",
+    "\n",
+    "  - ``table1`` and ``table2`` are in a ``1:1`` relationship\n",
+    "  - The key of ``table1`` is contained in that of ``table2`` (ie. keys are hierarchical)\n",
```
Comment (folmos-at-orange) [inline Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: I'd add some small changes:
- "The first two values (`str`) ... " 
- "A third value (`bool`) can be optionally set as `True` to indicate that the relation is `1:1`.
- "For example, **If** the ..." -> "For example, **if** the ..." (lower case)

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb [hunk 2]
```diff
@@ -101,14 +109,16 @@
...
+    "\n",
+    "If the ``relations`` field is not present then Khiops Python assumes that the tables are in a *star* schema."
```
Comment (folmos-at-orange) [inline Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: I'd add : "... *star* schema with `main_table` as the central table."

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb [hunk 3]
```diff
@@ -36,17 +36,20 @@
...
     "\n",
     "To train the KhiopsClassifier for this setup, we must specify a multi-table dataset. Let's first check the content of the tables:\n",
     "\n",
-    "- The main table `Accidents`\n",
-    "- The secondary table `Vehicles`\n",
-    "- The tertiary table `Users`\n"
+    "- The main table `Accidents`.\n",
+    "- The first secondary table `Vehicles` which has a 1:n relationship with `Accidents`.\n",
+    "- The second secondary table `Places` which has a 1:1 relationship with `Accidents`.\n",
+    "- The tertiary table `Users` which has a 1:n relationship with `Vehicles`.\n"
```
Comment (folmos-at-orange) [inline Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: I'd put `1:1` with backticks ``` `1:1` ```. Similarly with `1:n`

**Review by folmos-at-orange — CHANGES_REQUESTED** [review]
Minor stuff.

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb (hunk 3 above)
Comment (nairbenrekia) [reply Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: Ok.

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb (hunk 2 above)
Comment (nairbenrekia) [reply Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: Ok.

File: Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb (hunk 1 above)
Comment (nairbenrekia) [reply Sklearn Basics 3 - Train a Classifier on a Snowflake Multi-Table Dataset.ipynb]: Ok.

**Review by folmos-at-orange — APPROVED** [review]

**Pull request merged**